docs/reference/install.md

```diff
@@ -279,30 +76,12 @@ and their default values.
 | `imagePullSecrets`               | Names of image pull secrets to use                              | `dockerhub`                                            |
 | `replicas`                       | The number of replicas to run for the Crossplane operator       | `1`                                                    |
 | `deploymentStrategy`             | The deployment strategy for the Crossplane operator             | `RollingUpdate`                                        |
-| `clusterPackages.aws.deploy`       | Deploy AWS package                                                | `false`
-| `clusterPackages.aws.version`      | AWS provider version to deploy                                     | `<latest released version>`
-| `clusterPackages.gcp.deploy`       | Deploy GCP package                                                | `false`
-| `clusterPackages.gcp.version`      | GCP provider version to deploy                                     | `<latest released version>`
-| `clusterPackages.azure.deploy`     | Deploy Azure package                                              | `false`
-| `clusterPackages.azure.version`    | Azure provider version to deploy                                   | `<latest released version>`
-| `clusterPackages.rook.deploy`      | Deploy Rook package                                               | `false`
-| `clusterPackages.rook.version`     | Rook provider version to deploy                                    | `<latest released version>`
-| `personas.deploy`                | Install roles and bindings for Crossplane user personas         | `true`
-| `templateStacks.enabled`         | Enable experimental template stacks support                     | `true`
-| `templateStacks.controllerImage` | Template Stack controller image                                 | `crossplane/templating-controller:v0.2.1`
-| `priorityClassName      `        | Priority class name for crossplane and package manager pods       | `""`
+| `priorityClassName`        | Priority class name for crossplane and package manager pods       | `""`
 | `resourcesCrossplane.limits.cpu`        | CPU resource limits for Crossplane                       | `100m`
 | `resourcesCrossplane.limits.memory`     | Memory resource limits for Crossplane                    | `512Mi`
 | `resourcesCrossplane.requests.cpu`      | CPU resource requests for Crossplane                     | `100m`
 | `resourcesCrossplane.requests.memory`   | Memory resource requests for Crossplane                  | `256Mi`
-| `resourcesPackageManager.limits.cpu`      | CPU resource limits for PackageManager                     | `100m`
-| `resourcesPackageManager.limits.memory`   | Memory resource limits for PackageManager                  | `512Mi`
-| `resourcesPackageManager.requests.cpu`    | CPU resource requests for PackageManager                   | `100m`
-| `resourcesPackageManager.requests.memory` | Memory resource requests for PackageManager                | `256Mi`
```

Comment:
I missed putting some of the new values in here - we'll need to do that at some point. Can be a follow-on PR.

```yaml
rbacManager:
  deploy: true
  managementPolicy: All

resourcesRBACManager:
  limits:
    cpu: 100m
    memory: 512Mi
  requests:
    cpu: 100m
    memory: 256Mi
```

Reply:
@negz I can go ahead and add those here :+1: